app/controllers/nonsipp/membersurrenderedbenefits/SurrenderedBenefitsController.scala

```diff
@@ -26,7 +27,7 @@ import pages.nonsipp.membersurrenderedbenefits.SurrenderedBenefitsPage
 import play.api.data.Form
 import play.api.i18n.{I18nSupport, MessagesApi}
```

Comment:
now I18nSupport and FrontendBaseController imports are not used anymore as you changed them with PSRController. Can you please remove them ?